components/tours/crt_tour/crt_list_tutorial_tip.tsx

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react';
+import {FormattedMessage, useIntl} from 'react-intl';
+import {useSelector} from 'react-redux';
+
+import {GlobalState} from 'types/store';
+import {getIsMobileView} from 'selectors/views/browser';
+import {ShortcutKey, ShortcutKeyVariant} from 'components/shortcut_key';
+
+import {useMeasurePunchouts} from '@mattermost/components';
+
+import CRTTourTip from './crt_tour_tip';
+
+const translate = {x: -6, y: 62};
+
+export const UpShortcut = () => {
+    const {formatMessage} = useIntl();
+    return (<ShortcutKey variant={ShortcutKeyVariant.TutorialTip}>
+        {
+            formatMessage({
```

Comment:
@hmhealey Interesting, @pvev had the opposite opinion. Are there any performance differences? I liked `formatMessage` for the reason stated here https://github.com/mattermost/mattermost-webapp/pull/11108#discussion_r981017296 by Pablo or any future implications?

Reply:
I don't think there's any performance differences or anything major. I just prefer to use the JSX syntax for things whenever possible. That said, I don't feel strongly enough to change it.

I really need to get around to writing a recommended style guide for the web app so that we can avoid thrashing on stuff like this since I hate when reviews disagree, especially when it happens within the comments on a single PR 😅